(DOCSP-10287): Add Preview banner (#8)
mongodb/docs-mongodb-vscode: source/includes/fact-vsce-preview.rst

.. admonition:: Preview
   :class: note

   |vsce| is currently available as a **Preview** in the Visual Studio

>    Marketplace. The product and the corresponding documentation may
>    change at any time during the Preview stage.

Reviewer: ```suggestion
   Marketplace. The product, its features, and the corresponding documentation
   may change during the Preview stage.
```
I'd remove "at any time". Maybe it's because I am not a native speaker but it gives me the feeling that things will randomly change every other day. I don't have a strong opinion though, I trust you on this.
